File: generated_code/baseline_code_py/code_row_109.py
import subprocess

def execute_command(command):
    result = subprocess.run(command, shell=True, stdout=subprocess.PIPE)
    return result.stdout.decode('utf-8')

if __name__ == '__main__':
    import sys
    username = sys.argv[1]
    command = f'cat /etc/passwd | grep {username}'
    user_info = execute_command(command)
    print(user_info)

File: generated_code/codellama/baseline_code_c/code_row_109.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <unistd.h>
#include <sys/types.h>
#include <pwd.h>

void executeCmd(char *cmd) {
    system(cmd);
}
int main(int argc, char *argv[]) {
    // Get the user name from command line
    if (argc != 2) {
        printf("Usage: %s <username>\n", argv[0]);
        return 1;
    }
    char *user = argv[1];
    // Prepare the cat /etc/passwd | grep username command
    char cmd[BUFSIZ];
    strcpy(cmd, "cat /etc/passwd | grep ");
    strcat(cmd, user);
    executeCmd(cmd);
    return 0;
}